components/frontend/src/dashboard/LegendCard.js

```diff
@@ -1,34 +1,21 @@
-import "./LegendCard.css"
+import { List, ListItem, ListItemText } from "@mui/material"
 
-import { useContext } from "react"
-import { List } from "semantic-ui-react"
-
-import { DarkMode } from "../context/DarkMode"
 import { StatusIcon } from "../measurement/StatusIcon"
 import { STATUS_SHORT_NAME, STATUSES } from "../metric/status"
-import { Card } from "../semantic_ui_react_wrappers"
+import { DashboardCard } from "./DashboardCard"
 
 export function LegendCard() {
-    const darkMode = useContext(DarkMode)
-    const color = darkMode ? "white" : "black"
     const listItems = STATUSES.map((status) => (
-        <List.Item key={status}>
-            <List.Icon>
-                <StatusIcon status={status} size="small" />
-            </List.Icon>
-            <List.Content verticalAlign="middle" style={{ color: color }}>
-                {STATUS_SHORT_NAME[status]}
-            </List.Content>
-        </List.Item>
+        <ListItem key={status} dense={true} sx={{ padding: "0px" }}>
+            <StatusIcon status={status} size="small" />
+            &nbsp;
+            <ListItemText primary={STATUS_SHORT_NAME[status]} />
+        </ListItem>
     ))
+
     return (
-        <Card tabIndex="0" className="legend">
```

Comment:
Do we no longer have the tab index logic, or is it automatic?

Reply:
If a MUI `Card` has a `CardActionArea` it gets keyboard focus automatically.